.github/workflows/build.yml

```
@@ -34,41 +34,92 @@ jobs:
       fail-fast: false
       matrix:
         include:
+          - artifact_name: onnxruntime-win-x64
+            os: windows-2022
+            build_opts: --cmake_extra_defines CMAKE_SYSTEM_NAME=Windows CMAKE_SYSTEM_PROCESSOR=x86_64 --config Release --parallel --compile_no_warning_as_error --update --build --build_shared_lib
+            result_dir: build/Release
+            release_config: Release
+          - artifact_name: onnxruntime-win-x64-gpu
+            os: windows-2022
+            cuda_version: 11.8.0
+            cuda_sub_packages: '["cudart", "cuobjdump", "nvcc", "nvdisasm", "thrust", "cublas_dev", "cufft_dev", "curand_dev", "cusolver_dev", "cusparse_dev", "visual_studio_integration"]' # いくつかは不要かもしれない
+            cudnn_url: https://developer.download.nvidia.com/compute/redist/cudnn/v8.5.0/local_installers/11.7/cudnn-windows-x86_64-8.5.0.96_cuda11-archive.zip
+            build_opts: --cmake_extra_defines CMAKE_SYSTEM_NAME=Windows CMAKE_SYSTEM_PROCESSOR=x86_64 --config Release --parallel --compile_no_warning_as_error --update --build --build_shared_lib --use_dml --use_cuda --cuda_version 11.8
+            result_dir: build/Release
+            release_config: Release
+          - artifact_name: onnxruntime-win-x86
+            os: windows-2022
+            build_opts: --x86 --cmake_extra_defines CMAKE_SYSTEM_NAME=Windows CMAKE_SYSTEM_PROCESSOR=x86 --config Release --parallel --compile_no_warning_as_error --update --build --build_shared_lib
+            result_dir: build/Release
+            release_config: Release
+          - artifact_name: onnxruntime-linux-x64
+            os: ubuntu-20.04
+            build_opts: --cmake_extra_defines CMAKE_SYSTEM_NAME=Linux CMAKE_SYSTEM_PROCESSOR=x86_64 --config Release --parallel --compile_no_warning_as_error --update --build --build_shared_lib
+            result_dir: build
+            release_config: Release
+          - artifact_name: onnxruntime-linux-x64-gpu
+            os: ubuntu-20.04
+            cuda_version: 11.8.0
+            cuda_sub_packages: "[]"

```

linuxの方は何もいらないんですね。不思議。

---

`[]`がデフォルトであり、この場合デフォルトの範囲でインストールが行われるようです。CLI引数に展開されることを踏まえればそんなに変な挙動ではないかと。

---

なるほどです！！　ちょっとメモだけお願いできると･･･！
```suggestion
            cuda_sub_packages: "[]"  # 全部インストール
```
（win版も全部インストールで良いのでは説もありそう）

---

Windowsの方と合わせて、このように書いてみました。
[`2ac685f` (#26)](https://github.com/VOICEVOX/onnxruntime-builder/pull/26/commits/2ac685f5b065276ea225c4fcc48bee2ed350a4b8)